en/_posts/2015-02-14-how-to-run-bottle-uwsgi-nginx.markdown

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: "Running your Python Bottle application on Nginx using uWSGI"
+categories: development
+tags: guide, getting-started, python
+lang: en
+author: thoorium
+---
+
+[Bottle](http://bottlepy.org/) is a fast, simple and lightweight WSGI micro web-framework for [Python](https://www.python.org/). Coupled with [uWSGI](https://uwsgi-docs.readthedocs.org/en/latest/) and [Nginx](http://nginx.org/) bottle will reach high performances to serve and distribute your content.
+
+1. Installing Python, Bottle, uWSGI and Nginx
+===============================
+
+In this guide we will be using _Debian Wheezy_ as our host OS. Getting everything installed is easy thanks to the wondeful package manager.
+
+Everything can be installed in a single ``apt-get`` which is available below:
+
+```bash
+sudo apt-get install python2.7 python-bottle uwsgi uwsgi-plugin-python nginx
+```
+
+This will install Python, Bottle, uWSGI, the plugin uWSGI need to run Python applications and Nginx. If you plan to run your application using a Python virtual environnement, do not install ``python-bottle``.
+
+2. Hello Runabove
+===============
+
+For this exemple we will create a basic _Hello Runabove_ bottle application that goes like this:
+
+```python
+import bottle
+
+from bottle import route, template
+
+app = application = bottle.default_app()
```

Comment:
Could you add a comment to highlight this is where the magic happens ? Especially that "application" is *required* for WSGI to actually do anything. This will greatly help spring people like me headaches ;)